test/Microsoft.AspNetCore.Mvc.Core.Test/Builder/MvcAreaRouteBuilderExtensionsTest.cs

```diff
@@ -185,7 +185,7 @@ public void MapAreaRoute_DefaultsConstraintsAndDataTokens()
         }
 
         [Fact]
-        public void MapAreaRoute_ReplacesValuesForArea()
+        public void MapAreaRoute_NotReplaceValuesForAreaIfAlredyPresentInConstraintsOrDefaults()
```

Comment:
Ok renamed to MapAreaRoute_DoesNotReplaceValuesForAreaIfAlreadyPresentInConstraintsOrDefaults.